bukuserver/views.py

```diff
@@ -483,17 +477,15 @@ def get_list(
         filters: List[Tuple[int, str, str]],
         page_size: int = None,
     ) -> Tuple[int, List[types.SimpleNamespace]]:
-        logging.debug("search: %s", search)
+        app.logger.debug('search: %s', search)
+        if (arrow.now() - self.refreshed).seconds > 60:  # automatic refresh if more than a minute passed since last one
+            self._refresh()
+        else:
+            app.logger.debug('Tags cache refreshed %ss ago', (arrow.now() - self.refreshed).seconds)
```

Comment:
Ensured that tags data does not get more stale than 1 minute (with some logging when in debug mode)